rom/merge.cpp

```diff
@@ -306,6 +306,8 @@ void GetParametricTimeWindows(const int nset, const bool SNS, const std::string&
         double overlapMidpoint = (windowLeft + windowRight) / 2;
         twep.Append(overlapMidpoint);
 
+        cout << windowLeft << " " << windowRight << endl;
```

Comment:
Is this just for debugging, or is it generally useful? If it should be kept, is it clear what the output is (no label in the output)?

Reply:
This is just for debugging. I should remove it.